src/Symfony/Component/Messenger/Transport/AmqpExt/Connection.php

```diff
@@ -81,29 +93,19 @@ class Connection
      *   * delay:
      *     * routing_key_pattern: The pattern of the routing key (Default: "delay_%routing_key%_%delay%")
      *     * queue_name_pattern: Pattern to use to create the queues (Default: "delay_queue_%routing_key%_%delay%")
-     *     * exchange_name: Name of the exchange to be used for the retried messages (Default: "retry")
+     *     * exchange_name: Name of the exchange to be used for the retried messages (Default: "delay")
      *   * auto_setup: Enable or not the auto-setup of queues and exchanges (Default: true)
-     *   * loop_sleep: Amount of micro-seconds to wait if no message are available (Default: 200000)
      *   * prefetch_count: set channel prefetch count
      */
-    public function __construct(array $connectionOptions, array $exchangeOptions, array $queuesOptions, AmqpFactory $amqpFactory = null)
```

Comment:
I moved the options phpdoc from constructor to `fromDsn` because the constructor does have different parameters for different options (exchange, queue). So the options doc there makes no sense.